frontend/app/src/store/settings/general.ts

```diff
@@ -62,7 +62,7 @@ export const useGeneralSettingsStore = defineStore('settings/general', () => {
   const floatingPrecision: ComputedRef<number> = computed(() =>
     get(uiFloatingPrecision)
```

Comment:
This is also a doubly wrapped computed right?

Reply:
Oops I missed that